IdentityServer/v7/docs/content/tokens/refresh.md

```diff
@@ -47,34 +47,33 @@ var response = await client.RequestRefreshTokenAsync(new RefreshTokenRequest
 
 The [Duende.AccessTokenManagement](https://github.com/DuendeSoftware/Duende.AccessTokenManagement/wiki) library can be used to automate refresh & access token lifetime management in ASP.NET Core.
 
-## Refresh token security considerations
+## Binding refresh tokens
 Refresh tokens are a high-value target for attackers, because they typically have a much higher lifetime than access tokens.
 
-It is recommended that a refresh token is either bound to the client via a client secret (for confidential/credentialed clients), or rotated for public clients.
+However, refresh tokens for _confidential clients_ are bound to that client; they can only be used by the client they are issued to and the client is required to authenticate itself in order to do so. An attacker able to obtain a refresh token issued to a confidential client cannot use it without the client's credentials.
 
-The following techniques can be used to reduce the attack surface of refresh tokens.
+Refresh tokens issued to public clients are not bound to the client in the same way since the client cannot authenticate itself. Instead, we recommend that such refresh tokens be sender-constrained using [Proof of Possession]({{< ref "pop" >}}).
```

Comment:
```suggestion
Refresh tokens issued to public clients are not bound to the client in the same way, since the client cannot authenticate itself. We recommend that such refresh tokens be sender-constrained using [Proof of Possession]({{< ref "pop" >}}) instead.
```